src/opendr/perception/activity_recognition/x3d/x3d_learner.py

```diff
@@ -116,7 +115,7 @@ def __init__(
         self.seed = seed
         self.num_classes = num_classes
         self.loss = loss
-        self.ort_session = None
+        self._ort_session = None
```

Comment:
On unchanged line 129 below, "`self.backbon`e" ->  "`self.backbone` "

Reply:
Fixed in cce42a75